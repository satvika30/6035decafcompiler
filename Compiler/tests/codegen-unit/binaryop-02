class Program {
    void main() {
   		int a, b, c, d, e ,f;
        a = 70 + 2;
        b = 7;        
        c = a + (-62) + b;
        callout("printf", "%d\n", c);
        f = b - c + a; 
        callout("printf", "%d\n", f);
    }
}
