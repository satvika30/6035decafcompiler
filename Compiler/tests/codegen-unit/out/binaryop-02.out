72
81
26
62
-150